src/api/ZoweExplorerZosmfApi.ts

```diff
@@ -41,17 +41,19 @@ class ZosmfApiCommon implements ZoweExplorerApi.ICommon {
         return this.session;
     }
 
-    public async getStatus(validateProfile?: IProfileLoaded, profileType?: string): Promise<boolean> {
+    public async getStatus(validateProfile?: IProfileLoaded, profileType?: string): Promise<string> {
         // This API call is specific for z/OSMF profiles
         if (profileType === "zosmf") {
             const validateSession = await zowe.ZosmfSession.createBasicZosmfSession(validateProfile.profile);
             const sessionStatus= await zowe.CheckStatus.getZosmfInfo(validateSession);
 
             if (sessionStatus) {
-                return true;
+                return "active";
             } else {
-                return false;
+                return "inactive";
             }
+        } else {
+            return "unverified";
```

Comment:
@jellypuno this is nice but this is still being handled on the extender's side. When I try a different extension's profile that may not yet have the new getStatus() method it breaks like I mentioned above. An extender would have to make sure they have this method in place on their side and tested before this is released, and release their update right after this goes out

Reply:
@JillieBeanSim Thank you for testing it. I will take a look at it.

Reply:
@JillieBeanSim I added some changes that I think could fix this issue. Can you please verify? Thank you!